lib/taskana-core/src/test/java/acceptance/task/transfer/TransferTaskAccTest.java

```diff
@@ -435,4 +437,40 @@ void should_NotSetTheTransferFlagWithinBulkTransfer_When_SetTransferFlagNotReque
 
     assertThat(transferredTasks).extracting(TaskSummary::isTransferred).containsOnly(false);
   }
+
+  @WithAccessId(user = "teamlead-1")
+  @Test
+  void should_SetOwnerAndNotBeAuthorized_When_TransferringTask() throws Exception {
```

Comment:
What do you test here? Why does changing the owner change the permissions? They should be only dependant on the current user in all of the cases I know. It would be also better to not use transferWithOwner here two times, the first one untested

Reply:
Why does the test name include "NotAuthorized"? The test seems to test the happy path for transferring a Task with owner